docs/download_files.md

```diff
@@ -95,32 +95,51 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
 
 ## Portal-wide downloads
 
-<!--TODO: Fill in information about portal-wide downloads and provide screenshots of all file structure downloads -->
+The Portal-wide download page can be used to instantly download all [metadata](#metadata-only-downloads) or gene expression data for all samples on the Portal. 
```

Comment:
I also changed instantly --> at once, since it will not be instant 😬 
```suggestion
The Portal-wide Download page can be used to download all [metadata](#metadata-only-downloads) or gene expression data for all samples on the Portal at once. 
```